### ISSUE TITLE
crafting 

### ISSUE THREAD

**Issue by MachadoGT** [overextended/ox_inventory#1242]
**Is your feature request related to a problem? Please describe.**

Good evening dear friend, I have a question regarding the ox_inventory crafting system, which code is used so that only organizations can open the inventory? because I saw in the manual that you can add it but I don't know how to do it, can you tell me how to do it?

**Describe the solution you'd like**
how to use groups in craft
groups: table
Key-value pairs of job name and minimum grade to access the crafting location.
{["police"] = 0, ["ambulance"] = 2}

ok basically i tried but i can't if you can tell me how to limit the craft to some organizations or send me the code snippet so i can do it i would be very grateful
https://overextended.github.io/docs/ox_inventory/Guides/crafting

**Comment by thelindat** [comment]
So you're acknowledging the feature exists, and that the docs say how to use it, but then marking this as "enhancement" as though it's a feature request?
